packages/host/package.json

```diff
@@ -4,17 +4,19 @@
   "main": "./src/index.ts",
   "private": true,
   "devDependencies": {
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.0-beta.8",
```

Comment:
this failed to install with yarn and causes issues. hoping for a release soon, so we can bump version.
For now, you could up it to `^0.5.0-rc.5`